bin/pygrb/pycbc_make_offline_grb_workflow

```diff
@@ -130,7 +129,7 @@ else:
 
 sciSegs = segmentlistdict(sciSegs)
 if onSrc is None:
-    plot_met = make_grb_segments_plot(
+    make_grb_segments_plot(
```

Comment:
It looks like this same change needs to be applied also a few lines below, in the `else:`, but then the line
`    segs_plot = _workflow.File(plot_met[0], plot_met[1], plot_met[2],
                               file_url=plot_met[3])` needs to be updated.

Reply:
Ah, you did not change the definition of `make_grb_segments_plot`, which still ends with `return [ifos, plot_name, extent, plot_url]`.  Isn't `_ =  make_grb_segments_plot(...)` cleaner where you made the changes?

Reply:
The point of what I changed here is that the variable `plot_met` is unnecessary in those lines, because the `make_grb_segments_plot()` call is followed almost immediately by `sys.exit()` and nothing is done with `plot_met` in those cases.

In the `else:` block, `plot_met` *is* used (as you pointed out), so I have not changed that call.

`_ = make_grb_segments_plot()` is not necessary (a function that returns something can still be called without assigning its return value to anything) and I think omitting the assignment is simpler.

Reply:
In fact, the `else:` block is actually pointless, we could just remove the `else:` and unindent the content.